.gitlab-ci.yml

```diff
@@ -450,3 +450,127 @@ publish:mongodb-backup-runner:
+    TEST_QEMUX86_64_BIOS_GRUB: "false"
+    BUILD_QEMUX86_64_BIOS_GRUB_GPT: "false"
+    TEST_QEMUX86_64_BIOS_GRUB_GPT: "false"
+    BUILD_VEXPRESS_QEMU_UBOOT_UEFI_GRUB: "false"
+    TEST_VEXPRESS_QEMU_UBOOT_UEFI_GRUB: "false"
+    BUILD_VEXPRESS_QEMU: "false"
+    TEST_VEXPRESS_QEMU: "false"
+    BUILD_VEXPRESS_QEMU_FLASH: "false"
+    TEST_VEXPRESS_QEMU_FLASH: "false"
+    BUILD_BEAGLEBONEBLACK: "false"
+    BUILD_RASPBERRYPI3: "false"
+    BUILD_RASPBERRYPI4: "false"
+
+trigger:backend-integration-testing:mender-qa:
+  extends: .template:trigger:backend-integration-testing
+  trigger:
+    project: Northern.tech/Mender/mender-qa
+    strategy: depend
+
+trigger:acceptance-testing:inventory-enterprise:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/inventory-enterprise
+    strategy: depend
+
+trigger:acceptance-testing:mtls-ambassador:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/mtls-ambassador
+    strategy: depend
+
+trigger:acceptance-testing:mender-gateway:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/mender-gateway
+    strategy: depend
+
+trigger:acceptance-testing:deployments-enterprise:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/deployments-enterprise
+    strategy: depend
+
+trigger:acceptance-testing:useradm-enterprise:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/useradm-enterprise
+    strategy: depend
+
+trigger:acceptance-testing:auditlogs:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/auditlogs
+    strategy: depend
+
+trigger:acceptance-testing:deviceconfig:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/deviceconfig
+    strategy: depend
+
+trigger:acceptance-testing:deviceauth:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/deviceauth
+    strategy: depend
+
+trigger:acceptance-testing:deviceauth-enterprise:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/deviceauth-enterprise
+    strategy: depend
+
+trigger:acceptance-testing:inventory:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/inventory
+    strategy: depend
+
+trigger:acceptance-testing:tenantadm:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/tenantadm
+    strategy: depend
+
+trigger:acceptance-testing:useradm:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/useradm
+    strategy: depend
+
+trigger:acceptance-testing:deployments:
+  extends: .template:trigger:acceptance-testing
+  trigger:
+    project: Northern.tech/Mender/deployments
+    strategy: depend
```

Comment:
I think we cannot do this. 13 pipelines extra on every build of `mender-test-containers` is just too expensive for its value. Think that this will run on updates on gui testing image, client images or mender-dist-packages images.

Alternatives that I can think of:
* limit it to 3 or 4 repos, the ones we know use the most dependencies
* make the triggers manual, and run them only when we are updating the backend acceptance tests Dockerfile

Reply:
okey I do not like the manual way: there is no capacity to remember about doing it. I vote for 4 repos. let me choose them.